catalog/app/utils/BucketConfig.ts

```diff
@@ -0,0 +1,52 @@
+import * as R from 'ramda'
+import * as React from 'react'
+import * as redux from 'react-redux'
+import * as urql from 'urql'
+
+import * as AuthSelectors from 'containers/Auth/selectors'
+import * as Config from 'utils/Config'
+import * as NamedRoutes from 'utils/NamedRoutes'
+import { useRoute } from 'utils/router'
+
+import BUCKET_CONFIGS_QUERY from './BucketConfigList.generated'
+
+// always suspended
```

Comment:
If this comment is for me too, I don't understand it

Reply:
previously, this hook had a `suspend` parameter, which allowed to opt-out of using suspense by setting it to false, but since its creation this feature hasnt been used, so i removed this option and made it always use suspense